charts/dex-2.39.1/README.md

```diff
@@ -56,6 +55,7 @@ but by far the most popular and portable is undoubtedly [Cert Manager](https://c
 Cert Manager can be [installed](https://cert-manager.io/docs/installation/kubernetes) with a few steps:
 
 ```shell
+ocne application install --namespace cert-manager --release cert-manager --name cert-manager
 helm repo add jetstack https://charts.jetstack.io
```

Comment:
Should we add instructions on how to do this using ocne a well?

Reply:
To add the jetstack chart repo?  No.  We can install cert-manager using OCNE with the command that I added.  I have removed the references to helm commands, since they are not applicable.